docs/misc/dvd-remux.md

```diff
@@ -0,0 +1,670 @@
+                ```
+
+                Replace the following keys:
+
+                - `<input_file>`: The path to your DVDISO
+                - `<title>`: The number of the title you want to remux (integer between 1 and 99)
+                - `<output_file>`: The path to the output file
+
+            === "Title with multiple episodes"
+
+                ```bash
+                ffmpeg -f dvdvideo -preindex True -title <title> -chapter_start <chapter_start> -chapter_end <chapter_end> -i "<input_file>" -map 0 -c copy "<output_file>"
+                ```
+
+                Replace the following keys:
+
+                - `<input_file>`: The path to your DVDISO
+                - `<title>`: The number of the title you want to remux (integer between 1 and 99)
+                - `<chapter_start>`: The number of the chapter to start at (inclusive)
+                - `<chapter_end>`: The number of the chapter to end at (inclusive)
+                - `<output_file>`: The path to the output file
+
+            === "Title with a different angle"
+
+                ```bash
+                ffmpeg -f dvdvideo -preindex True -title <title> -angle <angle> -i "<input_file>" -map 0 -c copy "<output_file>"
+                ```
+
+                Replace the following keys:
+
+                - `<input_file>`: The path to your DVDISO
+                - `<title>`: The number of the title you want to remux (integer between 1 and 99)
+                - `<angle>`: The number of the angle you want to remux (integer between 1 and 9)
+                - `<output_file>`: The path to the output file
+
+            === "Menus"
+
+                ```bash
+                ffmpeg -f dvdvideo -menu True -pgc <pgc> -i "<input_file>" -map 0 -c copy "<output_file>"
+                ```
+
+                Replace the following keys:
+
+                - `<input_file>`: The path to your DVDISO
+                - `<pgc>`: The number of the PGC you want to remux (integer between 1 and 99)
+                - `<output_file>`: The path to the output file
+
+        !!! warning "Converting PCM audio to FLAC"
+
+            ??? question "Why is this necessary?"
+                DVD PCM audio uses big-endian byte order,
+                while WAV files use little-endian.
+                FFmpeg,
+                at the time of writing,
+                does not automatically handle this,
+                so we need to do it manually ourselves.
+
+                Since PCM is already raw,
+                it makes sense to convert it to FLAC
+                which will maintain the quality
+                while also ensuring proper byte order.
+                You can also opt to convert it to a lossy audio format,
+                such as AAC or Opus,
+                should you not want to use FLAC.
+
+            If your DVDISO has PCM audio
+            (which you can figure out
```

Comment:
Maybe add the example call here that was in the thewiki.moe guide. If you're worried about having too much content in this tab, this is another thing that could go outside of the tool tabs since it's about finding out what you're working with rather than the actual remuxing.